stats.tex

```diff
@@ -0,0 +1,50 @@
+%% Stats
+The \verb|sympy.stats| module provides random variable types and methods for
+determination of statistical properties of expressions involving random
+variables. The variables can be either continuous or discontinuous, which are
+further divided into finite and infinite. They are associated
+with probability densities on corresponding domains and internally defined
+in terms of probability spaces. 
+Apart from the possibility of defining the random variables from user supplied
+density distribution, SymPy provides definitions of most common
+distributions, including \texttt{Uniform}, \texttt{Poisson}, \texttt{Normal},
+\texttt{Binomial}, \texttt{Bernoulli}, and many others.
+
+Properties of random expressions can be calculated using, e.g.,
+\texttt{E} and \texttt{variance} to calculate expectation and variance.
+Conditions on random variables can be defined with inequalities, equalities,
+and logical operators and their overall probabilities are obtained using
+\texttt{P}. The features can be illustrated on a model of two dice throws:
+\begin{verbatim}
+>>> from sympy.stats import Die, P, E
+>>> X, Y = Die("X"), Die("Y")
+>>> P(Eq(X, 6) & Eq(Y, 6))
+1/36
+>>> P(X>Y)
+5/12
+\end{verbatim}
+The conditions can also be supplied as a second parameter to \texttt{E},
+\texttt{P}, and other methods to calculate the property given the condition:
+\begin{verbatim}
+>>> E(X, X+Y<5)
+5/3
+\end{verbatim}
+
+Using the facilities of the \texttt{sympy.stats} module, one can, for
+example, calculate 
+%Using the above-mentioned features, it is easy to calculate, for example
+the well known properties of maxwellian velocity distribution
+\begin{verbatim}
+>>> from sympy.stats import Maxwell, density
+>>> kT, m, x = symbols("kT m x", positive=True)
+>>> v = Maxwell("v", sqrt(kT/m))
+>>> E(v)                                # mean velocity
+2*sqrt(2)*sqrt(kT)/(sqrt(pi)*sqrt(m))
+>>> E(m*v**2/2)                         # mean energy
+3*kT/2
+>>> solve(density(v)(x).diff(x), x)[0]  # most probable velocity
+sqrt(2)*sqrt(kT)/sqrt(m)
+\end{verbatim}
+
+More information on the \texttt{sympy.stats} module can be found in
+\cite{StatsMRocklin}.
```

Comment:
@Upabjojr could you please explain what you mean exactly by rendering style?

In the version I'm looking at I see "can be found in [36]".  That's exactly how I saw citations rendered in very many articles and it looks absolutely natural to me (except that I'd probably add a non-breaking space `~` between "in" and `\cite`.

@rouckas I've never seen "in Ref. \cite" and I find it weird (but then YMMV).  "In the paper by \cite" is even weirder to me, because [36] is a paper, and not a list of authors.  "In the paper \cite" is something see just as often as "in \cite".